packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -97,22 +103,29 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
                 if (!resource) {
                     throw new Error(`${VscodeCommands.OPEN.id} command requires at least URI argument.`);
                 }
+
                 if (!URI.isUri(resource)) {
                     throw new Error(`Invalid argument for ${VscodeCommands.OPEN.id} command with URI argument. Found ${resource}`);
                 }
 
-                let options: TextDocumentShowOptions | undefined;
-                if (typeof columnOrOptions === 'number') {
-                    options = {
-                        viewColumn: columnOrOptions
-                    };
-                } else if (columnOrOptions) {
-                    options = {
-                        ...columnOrOptions
-                    };
+                if (await this.shouldCreateBeforeOpen(new TheiaURI(resource))) {
```

Comment:
I don't think we need to convert the URI here. The only stuff we need is #scheme and #toString(), both of which are availble on the URI type.

Reply:
Yes, you're right.
I changed the argument of `shouldCreateBeforeOpen` to be `URI` instead of `TheiaURI` in updated PR